src/status_im/multiaccounts/login/core.cljs

```diff
@@ -63,7 +63,7 @@
 (re-frame/reg-fx
  ::login
  (fn [[key-uid account-data hashed-password]]
-   (status/login key-uid account-data hashed-password)))
+   (status/login-with-config key-uid account-data hashed-password nil)))
```

Comment:
we don't use config in this PR ?

Reply:
i didn't find the config we need use, any suggestion ? @flexsurfer

Reply:
i think we should use same config no ? @cammellos